modules/standard/raid/raid_loot.sql

```diff
@@ -424,4 +424,162 @@ INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Py
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Aztur the Immortal', 1, 'Envoy to Chaos', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Aztur the Immortal', 1, 'Obsidian Desecrator', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Aztur the Immortal', 1, 'Ring of Blighted Flesh', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Aztur the Immortal', 1, 'Ring of Purifying Flame', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Aztur the Immortal', 1, 'Sacred Chalice', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Aztur the Immortal', 1, 'Sacred Text of the Immortal One', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Aztur the Immortal', 1, 'Sanguine Vambrace', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Aztur the Immortal', 1, 'Strength of the Immortal', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Aztur the Immortal', 1, 'Tattered book: Stampede of the Boar (section 5)', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Aztur the Immortal', 1, 'Tattered book: Stampede of the Boar (section 6)', '', 1);
+
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Khalum the Weaver of Flesh', 1, 'Fist of Heavens', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Khalum the Weaver of Flesh', 1, 'Summoner''s Staff of Dismissal', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Khalum the Weaver of Flesh', 1, 'Tattered book: Sting of the Viper (section 6)', '', 1);
+
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Uklesh the Beguiling', 1, 'Fist of Heavens', '', 1);
+INSERT INTO raid_loot (raid, category, ql, name, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'Uklesh the Beguiling', 1, 'Uklesh''s Talon', '', 1);
+
+
+INSERT INTO raid_loot (raid, category, ql, name, ident, forced_icon, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'General', 1, 'Notum Ring of the Three', 305488, 286792, 'green', 1);
+INSERT INTO raid_loot (raid, category, ql, name, ident, forced_icon, comment, multiloot) VALUES ('Temple of Three Winds (HL)', 'General', 1, 'Notum Ring of the Three', 305488, 286793, 'red', 1);
```

Comment:
same item href as the green one; noted and fixed for next commit.

Reply:
once you fix the ident/high_id for this one, you shouldn't need the forced_icon field anymore